.github/scripts/validate_cpu.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+import tensorflow as tf
+
+
+def check_gpu_not_available():
+    num_gpus = len(tf.config.experimental.list_physical_devices("GPU"))
```

Comment:
```suggestion
    num_gpus = len(tf.config.list_physical_devices("GPU"))
```

This is my bad, I gave you outdated advice. I think according to docs this is no longer experimental?

2.3: https://www.tensorflow.org/versions/r2.3/api_docs/python/tf/config/list_physical_devices
2.7: https://www.tensorflow.org/api_docs/python/tf/config/list_physical_devices

Reply:
nb: haven't tested this myself, so probably worth a double-check